static/gsAdmin/components/users/userEmails.tsx

```diff
@@ -34,6 +57,25 @@ function UserEmails({Panel, user}: Props) {
                   data.is_verified ? 'Verified' : 'Unverified'
                 }
               </td>
+              <td>
+                {data.email !== primary && (
+                  <Confirm
+                    message={`Are you sure you want to remove the email ${data.email}?`}
+                    onConfirm={() => handleRemoveEmail(data.email)}
+                    priority="danger"
+                  >
+                    <Button
+                      data-test-id="remove-email-button"
+                      priority="danger"
```

Comment:
not sure if there's a design pattern for this but i would just make the `priority="default"` for the buttons in the table, this is a random example i found from `settings/account/identities/ `

<img width="144" alt="Screenshot 2025-06-23 at 2 11 18 PM" src="https://github.com/user-attachments/assets/593325ff-c720-4298-82d2-0817e9902c3c" />